content/millennium/r4/general-clinical/condition.md

```diff
@@ -122,6 +167,7 @@ _Implementation Notes_
 * Only the body fields mentioned below are supported. Unsupported fields will be ignored.
 * Modifier fields should not be provided, and will cause the transaction to fail.
 * The code.coding field can have at most 2 codings, one of which must be set as userSelected true and the other one must be set as userSelected false.
+* Currently `problem-list-item` and `encounter-diagnosis` are supported.
```

Comment:
This line should be added in Search not in Create

Reply:
This is an extra line  for create currently, retrieve and search has three categories supported but create and update supoorts only two 
Update has the same line prior itself, just added the same here
<img width="780" alt="Screenshot 2020-12-10 at 4 22 31 PM" src="https://user-images.githubusercontent.com/3060097/101762879-fc5fa780-3b03-11eb-9d27-fea08d831358.png">